docs/source/api/plotting/index.rst

```diff
@@ -4,7 +4,6 @@ Plotting
 .. toctree::
 
   quick_plot
-  dynamic_plot
```

Comment:
Was this intentional?

Reply:
yes, the contents of that file are now in `quick_plot` since they go together